package.json

```diff
@@ -164,7 +167,8 @@
     "rimraf": "5.0.5",
     "semver": "7.6.0",
     "type-fest": "4.26.0",
-    "typescript": "5.3.3"
+    "typescript": "5.3.3",
```

Comment:
ideally we'd try to update TypeScript here but not strictly necessary to. Open to doing though.